tools/orxFontGen/build/premake4.lua

```diff
@@ -104,27 +104,18 @@ solution "orxFontGen"
 
     includedirs
     {
-        "../include",
-        "../../../code/include",
         "$(ORX)/include",
-        "../../../extern/stb_image",
-        "../../../extern/freetype/include"
+        "$(ORX)/../extern/stb_image",
+        "$(ORX)/../extern/freetype/include"
     }

```

Comment:
We need to keep both *static* paths and $(ORX)-based ones, otherwise the build machines will fail (cf. status of Travis-CI & AppVeyor).

Reply:
I changed the path to go back further in the tree. Will this affect Travis.-CI, or should ../include" be restored?